nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+reviewed: 2016-03-22
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Throughput on par with MSMQ
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Usage scenarios - deployment size
```

Comment:
Is our guidance really dependent on system size? I think customers might find it difficult to tell if their system is tiny, small, or rather big.

Also, they might start small and architect their system in a way that is not fit for growth they may not foresee.